source/frontend/src/hooks/useComposedProperties.ts

```diff
@@ -81,8 +82,14 @@ export const useComposedProperties = ({
         () => executeBcAssessmentSummary(retrievedPid),
         PROPERTY_TYPES.BC_ASSESSMENT,
       );
+      typeCheckWrapper(
+        () =>
+          executeGetAllFeatures({ PID: retrievedPid }, { forceSimplePid: true, timeout: 30000 }),
+        PROPERTY_TYPES.PARCEL_MAP,
+      );
+      typeCheckWrapper(() => findByPid(retrievedPid, true), PROPERTY_TYPES.PARCEL_MAP);
```

Comment:
@Maheshbabu28 could you please clarify if these two requests are duplicate? they seem to be. 
If not, please add a comment explaining why for other devs looking at this code - thanks!

Reply:
It's duplicate, removed(missed during rebase)